docs/index.md

```diff
@@ -6,180 +6,36 @@ nav_order: 0
 
-The Git-like interface means users of lakeFS can use the same development
-workflows for code and data.  Git workflows greatly improved software
-development practices; we designed lakeFS to bring the same benefits to
-data.
+## Same workflows for code and data
 
-In this way, lakeFS brings a unique combination of performance and manageability to data lakes. *To learn more about applying Git principles to data, [see here](https://lakefs.io/how-to-manage-your-data-the-way-you-manage-your-code/).*
+The Git-like interface means users of lakeFS can use the same development workflows for code and data. Git workflows greatly improved software development practices; we designed lakeFS to bring the same benefits to data.
 
-The open source lakeFS project supports AWS S3, Azure Blob Storage and Google Cloud Storage (GCS) as its underlying storage service. It is API compatible with S3 and integrates seamlessly with popular data frameworks such as Spark, Hive, dbt, Trino, and many others.
+In this way, lakeFS brings a unique combination of performance and manageability to data lakes. To learn more about applying Git principles to data, [see here](https://lakefs.io/how-to-manage-your-data-the-way-you-manage-your-code/).
 
-{: .pb-5 }
+## Supports the major cloud providers
 
+The open source lakeFS project supports AWS S3, Azure Blob Storage and Google Cloud Storage (GCS) as its underlying storage service. It is API compatible with S3 and integrates seamlessly with popular data frameworks such as Spark, Hive, dbt, Trino, and many others.
 
-## New! lakeFS Playground
+## NEW lakeFS Playground
 
-Experience lakeFS first hand with your own isolated environment.
-You can easily integrate it with your existing tools, and feel lakeFS in action in an environment similar to your own.
+Experience lakeFS first hand with your own isolated environment. You can easily integrate it with your existing tools, and feel lakeFS in action in an environment similar to your own.
 
-<p style="margin-top: 40px;">
-    <a class="btn btn-green" href="https://demo.lakefs.io/" target="_blank">
-        Try lakeFS now without installing
-    </a>
-</p>
+> [Try lakeFS now without installing](https://demo.lakefs.io/)
 
 ## How do I use lakeFS?
 
-lakeFS maintains compatibility with the S3 API to minimize adoption
-friction.  Use it as a drop-in replacement for S3 from the perspective of
-any tool interacting with a data lake.
+lakeFS maintains compatibility with the S3 API to minimize adoption friction. Use it as a drop-in replacement for S3 from the perspective of any tool interacting with a data lake.
 
 For example, take the common operation of reading a collection of data from object storage into a Spark DataFrame. For data outside a lakeFS repo, the code will look like:
 
-```py
-df = spark.read.parquet("s3a://my-bucket/collections/foo/")
-```
-
+> df = spark.read.parquet("s3a://my-bucket/collections/foo/")
```

Comment:
Why not a code block?

Reply:
this is not a tutorial page. It's an information page. Code blocks makes sense in a tutorial.